### PR TITLE
[OSS-3] Initial documentation files

### PR DESCRIPTION
**Pull request by DavidJGapCR**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+since many employees of one corporation may be contributing. To see the full
+list of contributors, see the revision history in source control or the [GitHub
+contributors
+graph](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Standard/graphs/contributors).
+New contributors should add their name and e-mail address or link to GitHub
+profile to this file with their first pull request.
+
+Ed-Fi Alliance\
+Mr. Ed <talkinghorse@ed-fi.org>\
+[Gumby](https://github.com/this-is-a-fake-profile)
```
Comment (lalrg) [inline CONTRIBUTORS.md:15]: I think yoy can remove from line 13 to 15 since this contributor is just an example

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Contributors List
+
+This is the list of Ed-Fi-ODS-Implementation's significant contributors.
+
+This does not necessarily list everyone who has contributed code, especially
+since many employees of one corporation may be contributing. To see the full
+list of contributors, see the revision history in source control or the [GitHub
+contributors
+graph](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Standard/graphs/contributors).
```
Comment (lalrg) [inline CONTRIBUTORS.md:9]: Please update this link to make it redirect to the current repository

File: NOTICES.md [hunk 3]
```diff
@@ -0,0 +1,1138 @@
...
+    copies or substantial portions of the Software.
+    
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+
```
Comment (lalrg) [inline NOTICES.md:427]: I think this file needs some format, you can look at [this review](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/1#discussion_r418247350) from Stephen for having a reference.

File: CONTRIBUTORS.md (hunk 1 above)
Comment (DavidJGapCR) [reply CONTRIBUTORS.md]: Fixed

File: CONTRIBUTORS.md (hunk 2 above)
Comment (DavidJGapCR) [reply CONTRIBUTORS.md]: Fixed

File: NOTICES.md (hunk 3 above)
Comment (DavidJGapCR) [reply NOTICES.md:423]: Fixed

**Pull request merged**